packages/server/graphql/types/JiraServerIssue.ts

```diff
@@ -96,14 +94,10 @@ const {connectionType, edgeType} = connectionDefinitions({
   nodeType: JiraServerIssue,
   edgeFields: () => ({
     cursor: {
-      type: GraphQLISO8601Type
+      type: GraphQLString
     }
   }),
   connectionFields: () => ({
-    pageInfo: {
```

Comment:
-1 why remove `pageInfo`? If the date cursor type isn't working, you can use the `PageInfo` type instead: https://github.com/ParabolInc/parabol/blob/master/packages/server/graphql/types/PageInfo.ts

Reply:
@nickoferrall I removed it here, because we have default one defined here https://github.com/ParabolInc/parabol/blob/8a848a90858a29870572161ee59586f1c2b1fb15/packages/server/graphql/connectionDefinitions.ts#L40